integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/altinn/MessagePolling.java

```diff
@@ -175,7 +175,7 @@ public void checkForNewMessages() throws MessageException {
                     Audit.info(format("Message id=%s is a receipt", eduDocument.getConversationId()),
                             eduDocument.createLogstashMarkers().and(getReceiptTypeMarker(jaxbKvit.getValue())));
                     MessageStatus status = statusFromKvittering(jaxbKvit.getValue());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "status" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "status". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1854)